runtime/tests/interpreter/string_test.go

```diff
@@ -115,7 +115,7 @@ func TestInterpretStringEncodeHex(t *testing.T) {
 
 	inter := parseCheckAndInterpret(t, `
       fun test(): String {
-          return String.encodeHex([1 as UInt8, 2, 3, 0xCA, 0xDE])
+          return String.encodeHex([1, 2, 3, 0xCA, 0xDE] as [UInt8])
```

Comment:
Like above in the checker test, maybe simplify this by removing the explicit type annotation, now that it is not necessary anymore:

```suggestion
          return String.encodeHex([1, 2, 3, 0xCA, 0xDE])
```